src/Meta/Base.php

```diff
@@ -42,6 +42,6 @@ public function getEndDelimiter()
      */
     public function getStartTokenRegex()
     {
-        return '{^(<\?(php)?\s+)|<%|(<\?xml[^>]+)}i';
+        return '{^(<\?(php)?\s+(?:declare\(\s*\w+\s*=\s*[\w\d\'"-]+\s*\);\s+)*)|<%|(<\?xml[^>]+)}is';
```

Comment:
This allows for any combination and style, whether you use one-line, multi line, spaces/no spaces 👍
I did a thorough check of the regex.